.github/dependabot.yml

```
@@ -0,0 +1,16 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "pub"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "pub"
+    directory: "/sweyer_plugin"
+    schedule:
+      interval: "daily"
```

```suggestion
      interval: "daily"

 - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "daily"

```
Let's also let it keep the actions in our CI up to date.

---

[Docs](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot#example-dependabotyml-file-for-github-actions)